src/cryptonote_basic/miner.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include <sstream>
 #include <numeric>
+#include <thread>
```

Comment:
probably should be using tools::get_max_concurrency() instead of using std::thread directly.